src/Microsoft.Health.Dicom.Api/Features/Audit/AuditLoggingFilterAttribute.cs

```diff
@@ -21,16 +20,15 @@ public class AuditLoggingFilterAttribute : Microsoft.Health.Api.Features.Audit.A
         {
         }
 
-        public override void OnActionExecuted(ActionExecutedContext context)
+        /// <summary>
+        /// Do nothing when execution finishes for a request.
+        /// LogExecuted will be called from AuditMiddleware.
+        /// AuditEgressLogger is responsible for taking care of this.
+        /// </summary>
+        /// <param name="context">On result executed context.</param>
+        public override void OnResultExecuted(ResultExecutedContext context)
```

Comment:
Do we even need this class implemented if we do not do anything?

Reply:
Based on the current implementation, we do need this. OnResultExecuted, by default, is printing the LogExecuted messages such that batch requests can be logged properly in FHIR. Since, we can completely rely on AuditLoggingMiddleware, we need to override this such that the same messages do not get logged twice.